types/react-dates/react-dates-tests.tsx

```diff
@@ -137,16 +137,16 @@ class DayPickerRangeControllerMinimumTest extends React.Component {
     }
 }
 
-
-
-
-
-
-
-
-
-
-
+class DayPickerSingleDateControllerMinimumTest  extends React.Component {
+    render() {
+         return <DayPickerSingleDateController
```

Comment:
Please ensure your new test passes by adding `DayPickerSingleDateController` to the set of members you are importing from `react-dates`.